srcpkgs/openscad/template

```diff
@@ -1,14 +1,25 @@
 # Template file for 'openscad'
 pkgname=openscad
 version=2021.01
-revision=3
+revision=4
 build_style=qmake
 hostmakedepends="bison flex pkg-config gettext qt5-qmake qt5-host-tools"
 makedepends="cgal-devel harfbuzz-devel opencsg-devel qscintilla-qt5-devel
- qt5-multimedia-devel libxml2-devel libzip-devel double-conversion-devel"
+ qt5-multimedia-devel libxml2-devel libzip-devel double-conversion-devel
+ $(vopt_if 3mf lib3mf-devel)"
 short_desc="Solid 3D CAD modeller"
 maintainer="Pierre Allegraud <pierre.allegraud@crans.org>"
 license="GPL-2.0-or-later"
 homepage="http://www.openscad.org"
 distfiles="http://files.openscad.org/${pkgname}-${version}.src.tar.gz"
 checksum=d938c297e7e5f65dbab1461cac472fc60dfeaa4999ea2c19b31a4184f2d70359
+
+pre_patch() {
+	if [ "$build_option_3mf" ]; then
+		vsed -i features/lib3mf.prf -e "s/lib3MF/lib3mf/g"
+	fi
+}
+
+build_options="3mf"
+desc_option_png="Enable support for 3MF models"
+build_options_default="3mf"
```

Comment:
why make this a build option?

also, these variables should go before the `pre_patch()` function

Reply:
Not having it is the behavior beforehand and the library is optional, so I figured it qualified as an optional feature.

Reply:
Fixed the positioning